lib/i18n/locale/fallbacks.rb

```diff
@@ -56,7 +56,7 @@ class Fallbacks < Hash
       def initialize(*mappings)
         @map = {}
         map(mappings.pop) if mappings.last.is_a?(Hash)
-        self.defaults = mappings.empty? ? [I18n.default_locale.to_sym] : mappings
+        self.defaults = mappings.empty? ? [] : mappings
```

Comment:
@tjoyal This PR did not update the documentation comment above, which still references `default_locale`.

Took me far too long to realize that `default_locale` was no longer getting used. 😅